src/mono/sample/wasm/wasm.mk

```diff
@@ -23,7 +23,7 @@ run-browser:
 		echo "The tool dotnet-serve could not be found. Install with: $(DOTNET) tool install --global dotnet-serve"; \
 		exit 1; \
 	else  \
-		$(DOTNET) serve -d bin/$(CONFIG)/AppBundle -p 8000; \
+		$(DOTNET) serve -d=bin/$(CONFIG)/AppBundle -p 8000; \
```

Comment:
nit: `$(DOTNET) serve -d=bin/$(CONFIG)/AppBundle -p=8000; \` or even `$(DOTNET) serve -d:bin/$(CONFIG)/AppBundle -p:8000; \` might be more consistent?